src/SuiteSparseMatrixCollection.jl

```diff
@@ -27,68 +21,36 @@ function __init__()
 end
 
 "Formats in which matrices are available."
-const ssmc_formats = ("MM", "RB", "mat")
+const ssmc_formats = ("MM", "RB")
 
 """
-    group_paths(matrices; format="MM")
+     fetch_ssmc(group::AbstractString, name::AbstractString; format="MM")
 
-Return the array of paths where `matrices`'s groups will be or were downloaded.
+Download the matrix with name `name` in group `group`.
+Return the path where the matrix is stored.
 """
-function group_paths(matrices; format="MM")
-  format ∈ ssmc_formats || error("unknown format $format")
-  joinpath.(ssmc_dir, format, matrices.group)
+function fetch_ssmc(group::AbstractString, name::AbstractString; format="MM")
+  group_and_name = group * "/" * name * "." * format
```

Comment:
Is it working on Windows ?
`\\` is used instead of `/` for this OS.

Reply:
That's not a path. It's the matrix's full name.